unattended_installer/install_functions/checks.sh

```diff
@@ -240,9 +240,10 @@ function check_dist() {
         fi
     fi
 
-    if [ -n "${notsupported}" ] && [ -z "${ignore}" ]; then
-        common_logger -e "The recommended systems are: Red Hat Enterprise Linux 7, 8, 9; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04, 22.04. The current system does not match this list. Use -i|--ignore-check to skip this check."
-        exit 1
+    if [ -n "${notsupported}" ]; then
+        
```

Comment:
Remove this empty line (244), please.

Reply:
Fixed in [6bbaf9e](https://github.com/wazuh/wazuh-packages/pull/2991/commits/6bbaf9e6863e11e47e3f5d36c841f7d0e45a7d6c)